motioneye/mjpgclient.py

```diff
@@ -179,9 +183,11 @@ def _on_connect(self, future: Future) -> None:
             elif (
                 self._auth_mode == 'digest'
             ):  # in digest auth mode, the header is built upon receiving 401
+                logging.debug("digest authentication _on_connect")
                 self.write(b'GET / HTTP/1.0\r\n\r\n')
 
             else:  # no authentication
+                logging.debug(f"no authentication _on_connect")
```

Comment:
No need to use an f-string, is there is nothing to format 🙂:
```suggestion
                logging.debug('no authentication _on_connect')
```